src/main/java/goodgenerator/blocks/tileEntity/LargeEssentiaSmeltery.java

```diff
@@ -409,8 +409,8 @@ public void onPostTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
 
     @Override
     public boolean onRunningTick(ItemStack aStack) {
-        this.nodePurificationEfficiency = Math.max(0, this.nodePurificationEfficiency - 5);
-        if (xstr.nextInt(20) == 0) {
+        this.nodePurificationEfficiency = Math.max(0, this.nodePurificationEfficiency - 1;
+        if (xstr.nextInt(20) == 0) {)
```

Comment:
wrong parenthesis placement

Reply:
> also what makes you think the values are incorrect?

The node power consumed during work is always more than the absorbed energy